app/models/conversion_host.rb

```diff
@@ -34,6 +34,7 @@ def source_transport_method
   end
 
   def ipaddress(family = 'ipv4')
+    return resource.hostname if resource.hostname
```

Comment:
Hm I wouldn't expect an ipaddress method to return a dns hostname, maybe another method that tries hostname then ipaddress?

Reply:
I've added a delegation for `hostname` and updated `miq_ssh_util_args` to use it if present.